src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -258,4 +258,12 @@ public function theCustomerServiceShouldKnowAboutThisAdditionalNotes(
 
         Assert::same($notes, $orderNotes);
     }
+
+    /**
+     * @Then /^(the administrator) should see that (order placed by "[^"]+") has "([^"]+)" currency$/
+     */
+    public function theAdministratorShouldSeeThatThisOrderHasBeenPlacedIn(): void
+    {
+        // Intentionally left blank, we do not return currency in API at the moment
```

Comment:
So IMO we should change the serialization of order to return currency code and to be able to check that

Reply:
We have a task for it in backlog, so should I change serialization here? or we change it in another task and put a note that this step should be implemented?

Reply:
In my opinion, we should cover all steps from scenarios that we are implementing in API context without leaving them working confusingly